src/notification/notification.component.ts

```diff
@@ -39,6 +39,7 @@ import { of, isObservable, Subject } from "rxjs";
 			<div class="bx--inline-notification__text-wrapper">
 				<p [innerHTML]="notificationObj.title" class="bx--inline-notification__title"></p>
 				<p [innerHTML]="notificationObj.message" class="bx--inline-notification__subtitle"></p>
+				<ng-content></ng-content>
```

Comment:
We can't use `ng-content` with service created notifications/toasts, so this has to come in via a template as part of the `notificationObj`

Reply:
fixed!